test/libraries/SwapFeeLibrary.t.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import "src/libraries/SwapFeeLibrary.sol";
+import "forge-std/Test.sol";
+
+contract SwapFeeLibraryTest is Test {
+    function test_isDynamicFee_returnsTrue() public {
+        uint24 dynamicFee = 0x800000;
```

Comment:
i know its tested in the fuzz but i would love to see this test also do `0xFFFFFF` is true. Like to show explicitly that only that 1 bit matters.

Reply:
gotcha - sure! fine to add non fuzzs on edge case values